ruby_event_store/lib/ruby_event_store/projection.rb

```diff
@@ -27,7 +27,7 @@ def init(handler)
 
     def when(events, handler)
       Array(events).each do |event|
-        handlers[event] = handler
+        handlers[event.to_s] = handler
```

Comment:
why `to_s`?

Reply:
Because I still want to setup projection with:

```
...
.when(SomeEventClass, -> { ... })
```

Reply:
What syntax do you imagine with `Proto`

Reply:
AFAIK `Proto` implements `type` - so you could use anything that matches Proto's events types.

Reply:
We could not use `event.class` anymore as we want to allow use self defined Event base classes.

Reply:
`to_s` actually allows Proto to be used here